.github/workflows/run-tests-on-modified-meta.yml

```diff
@@ -66,7 +81,13 @@ jobs:
         uses: actions/checkout@v4
         with:
           fetch-depth: 2
-
+      - name: Set RCLONE Service account env var from secret
+        shell: bash
+        run: |
+          echo "::add-mask::${{ needs.fetch-secret.outputs.gdrive_secret }}"
+          echo "RCLONE_CONFIG_MLC_COGNATA_SERVICE_ACCOUNT_CREDENTIALS=${{ needs.fetch-secret.outputs.gdrive_secret }}" >> $GITHUB_ENV
```

Comment:
These secrets are specific to some scripts. We should not add them in this generic test action. When such secrets are needed we should create a new github action specific to the script and run it on pull_request_target.